my-app/src/App/components/Multidropdown/MultiDropdown.module.scss

```
@@ -0,0 +1,98 @@
+.multiDropdown {
+    display: block;
+    &__button {

```

обычно оставляется пустая строка перед объявлением классов / селекторов
```suggestion
    display: block;
    
    &__button {
```